containers/java/clipper-java-container/src/test/java/com/clipper/container/app/ContainerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.clipper.container.app;
```

Comment:
Make a RunContainer class in main (not in test) that creates a Noop model and runs. Similar to the Noop model example in Python. Basically, just copy this into main so it can be built and run as a normal java app.

After this PR is merged, we can build a Docker image for the java container.